ocaml/xapi/helpers.ml

```diff
@@ -1736,6 +1736,12 @@ end = struct
           ; "TIMEOUTclose = 1"
           ; "options = CIPHER_SERVER_PREFERENCE"
           ; "sslVersion = TLSv1.2"
+          ; ""
+          ; "# xapi connections use SNI 'pool' to request a cert"
+          ; "[pool]"
+          ; "connect = 80"
```

Comment:
Isn't this only used when using stunnel as a client?

Reply:
All I can say is that this works and I also believe the parameter is required but I might be wrong.

Reply:
The manual is not very clear, but it looks like it's only used when initiating connections:
```
      connect = [HOST:]PORT                                                                                                                                   
           connect to a remote address

           If no host is specified, the host defaults to localhost.
       
           Multiple connect options are allowed in a single service section.

           If host resolves to multiple addresses and/or if multiple connect options are specified, then the remote address is chosen using a round-robin
           algorithm.
```

Reply:
Isn't `connect = x` used to say to stunnel 'please turn incoming HTTPS requests into HTTP requests, and send them to x'?

Reply:
This is what we want, right? Forwarding an HTTPS request to port 80 but unencrypted.